docker-compose.server.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.6"
+
+services:
+  web:
+    build: .
```

Comment:
server docker compose should never be building from anything -- there is no Dockerfile on the server 

it only pulls from images -- this should also be exactly the same as what's actually on the server

Reply:
weird this is exactly what was on the server 🤔

Reply:
that is wacky sighhh okay just leave it in haha i'll fix it when i look more into why our server one has this